Completed post/put requests, added sorting

cmoralespv opened the pull request:
Implemented the helpfulness handler
Implemented the report handler
Added sorting logic to reviews list get request

cmoralespv committed:
implemented put requests for helfulness and report, added sorting to get reviews list

jnbooth commented on src/api/routes/get.js:
```diff
@@ -9,18 +9,38 @@ router.get('/reviews/', (req, res) => {
+    } else if (option === 'helpful') {
+      sort = 'helpfulness DESC';
+    } else {
+      sort = 'date DESC, helpfulness DESC';
```
clever way to handle the typo issue

jnbooth commented on src/api/routes/post.js:
```diff
@@ -45,6 +50,30 @@ router.post('/reviews/', (req, res) => {
+
+  sql.query(
+    `UPDATE reviews SET reported = NOT reported
+    WHERE review_id = ${review_id};`)
```
Currently a security flaw

cmoralespv commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Implement Mark Review as Helpful](https://trello.com/c/Kh3Ghuqo/28-implement-mark-review-as-helpful)

cmoralespv commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Added sorting to list reviews get request and refactored pagination to be like the heroku server](https://trello.com/c/IUj0xXgh/33-added-sorting-to-list-reviews-get-request-and-refactored-pagination-to-be-like-the-heroku-server)
